test/foundry/FleekERC721/FleekERC721.ap.t.sol

```diff
@@ -0,0 +1,142 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.17;
+
+import "./TestBase.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
+import {FleekAccessControl} from "contracts/FleekAccessControl.sol";
+
+contract Test_FleekERC721_AccessPoint is Test_FleekERC721_Base {
+    using Strings for address;
+    uint256 internal tokenId;
+
+    function assertAccessPointJSON(
+        string memory accessPointName,
+        string memory _tokenId,
+        string memory score,
+        string memory nameVerified,
+        string memory contentVerified,
+        address owner
+    ) internal {
+        string memory current = CuT.getAccessPointJSON(accessPointName);
+        // prettier-ignore
+        string memory expectedJSON = string(abi.encodePacked('{"tokenId":', _tokenId, ',"score":', score, ',"nameVerified":', nameVerified, ',"contentVerified":', contentVerified, ',"owner":"', owner.toHexString(), '"}'));
+        assertEq(current, expectedJSON);
+    }
+
+    function setUp() public {
+        baseSetUp();
+        tokenId = mintDefault(deployer);
+    }
+
+    function test_addAccessPoint() public {
+        CuT.addAccessPoint(tokenId, "accesspoint.com");
```

Comment:
Should we expect by calling a read function on the token Id?

Reply:
Do you mean by calling `tokenURI(tokenId)`? By now it would return a Base64, what I think is hard to parse in Solidity. What would help here is adding the function that returns the Solidity object, as it is in #93. Maybe we can add this assertion after implementing this function.

Reply:
For now, we can just check if the access point is actually added and if it exists. To check the details, we can expose the function that is suggested in Issue #93 later.